src/style.css

```diff
@@ -0,0 +1,76 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: bold;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+.container {
+  position: relative;
+  margin: 20px auto;
+}
+
+.search {
+  outline: none;
+  width: 100%;
+  font-family: "Avenir", sans-serif;
```

Comment:
let's move `font-family` to `html `tag and use `inherit` in all selectors we need